src/app/components/crosssection/crosssection.component.ts

```diff
@@ -193,7 +195,7 @@ export class CrosssectionComponent implements AfterViewInit {
               'textstyle': {
                 'color': '#757575'
               },
-              'anchor': AnchorType.LeftCenter
+              'alignment': AnchorType.LeftCenter
```

Comment:
I'd delete it. anchor is not alignment

Reply:
but then the axis labels will not be displayed

Reply:
okay